qpmodel/Statis.cs

```diff
@@ -585,9 +585,17 @@ public class SysStats : SystemTable
     {
         readonly Dictionary<string, ColumnStat> records_ = new Dictionary<string, ColumnStat>();
 
-        public void ClearRecords()
+        public void RemoveRecords(string tabName)
         {
-            records_.Clear();
+            var table = Catalog.systable_.Table(tabName);
+            var columns = table.columns_;
+
+            List<ColumnStat> stats = new List<ColumnStat>();
+            foreach (var v in columns)
+            {
+                var colName = v.Value.name_;
+                records_.Remove(tabName + colName);
+            }
```

Comment:
In case not to clear some build-in table log, only delete table records related.